build/docker/spec/coveralls

```diff
@@ -19,6 +16,9 @@ debian_deps="$debian_deps python-pip"
 debian_deps="$debian_deps wget"
 pip_deps="$pip_deps cpp-coveralls"
 
+export CONFIGURE_FLAGS="--enable-coverage"
+export MAKE_CHECK_TARGET="check"
+
```

Comment:
Style: use ALL_UPPERCASE for overridable variables. Stop assuming `pkg_configure_flags` can be passed around as that is not what is going to happen in this specific case. Reduces complexity.